osgpkitools/cert_utils.py

```diff
@@ -50,10 +50,10 @@ def prompt_for_password(verify):
 
 class Csr(object):
 
-    KEY_LENGTH = 2048
+    KEY_LENGTH = 4096
     PUB_EXPONENT = 0x10001
 
-    def __init__(self, hostname, output_dir=None, altnames=None, location=None):
+    def __init__(self, hostname, output_dir=None, altnames=None, location=None, key_length = KEY_LENGTH):
```

Comment:
```suggestion
    def __init__(self, hostname, output_dir=None, altnames=None, location=None, key_length=KEY_LENGTH):
```

From PEP8 https://www.flake8rules.com/rules/E251.html